### PR TITLE
Week 1 HW

### PR DESCRIPTION
**Pull request by DAMakeAWay4**

### CODE REVIEW

File: Week 1/Dashley/Index.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Derrick Ashley</title>
+</head> 
+<body>
+    <h1>Derrick Ashley the developer </h1>
+
+
+    
+</body>
```
Comment (bclincy) [inline Week 1/Dashley/Index.html]: You content should bet between the body tags

File: Week 1/Dashley/Index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+</html>
+
+
+    <section>
```
Comment (bclincy) [inline Week 1/Dashley/Index.html:12]: Where is your closing blockquote?

File: Week 1/Dashley/Index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+</p>>
+</div>
+<p>Object Orientated Programming</p>
+    <p class="adef">
+        Object Orientated Programming is used for software devlopment to structure a software program into simple, reusable pieces of data.</p>
+    </div>
+</section>
+</body>
+</html>
+
```
Comment (bclincy) [inline Week 1/Dashley/Index.html]: Remove these extra lines between 47 and 97

File: Week 1/Dashley/WEEK1HW.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>Derrick Ashley</title>
+</head> 
+<body>
+    <h1>Derrick Ashley the developer </h1>
+
+</body>
```
Comment (bclincy) [inline Week 1/Dashley/WEEK1HW.html]: closeing the body tag is usually the end of the page

File: Week 1/Dashley/WEEK1HW.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    <header>
```
Comment (bclincy) [inline Week 1/Dashley/WEEK1HW.html:11]: All opened tags need a closed tag, there are a few like the image tag that does, but you don't have a closing header tag.

**Review by DestianaC — APPROVED** [review]

**Review by bclincy — APPROVED** [review]

**Pull request merged**